core/blockchain/impl/block_tree_impl.cpp

```diff
@@ -231,8 +67,8 @@ namespace kagome::blockchain {
     // find epoch digest. First found digest if it in the block assigned to the
     // current epoch will be saved as digest planned for next epoch. First found
     // digest if it in the block assigned to the early epoch will be saved as
-    // digest for current epoch (and planned for next epoch, if it not defined
-    // yet).
+    // digest for current epoch (and planned for next epoch,
```

Comment:
Why?

Reply:
There was a typo (lacking 'is'), so I added is and also replaced "not defined" with "undefined", which supposedly cause clang-format to shift the end of the sentence to the next line. What exactly is the source of confusion here?